datareduction/datareductionoperator.h

```diff
@@ -383,40 +383,6 @@ namespace DRO {
 
    };
 
-   class MaxDistributionFunction: public DataReductionOperator {
```

Comment:
These are gone, is the list in parameters.cpp updated?

Reply:
yes